pyproject.toml

```diff
@@ -2,24 +2,32 @@
 requires = ["poetry-core>=1.1.0a6"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.mypy]
+python_version = 3.9
+show_error_context = true
+pretty = true
+show_column_numbers = true
+warn_return_any = true
```

Comment:
It's to help produce more helpful error messages when mypy encounters an error. When I first ran pre-commit, mypy found a bunch of type errors but the error messages were helpful. I wanted more detail, specifically where in the codebase the errors were so that I can quickly go and fix them. 

By performance hit, do you mean it might take longer for mypy to execute because of these additional configurations?